src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -36,7 +36,7 @@ const BASE_URL = 'hosts';
   providers: [{ provide: URLBuilderService, useValue: new URLBuilderService(BASE_URL) }]
 })
 export class HostsComponent extends ListWithDetails implements OnInit {
-  @ViewChild(TableComponent, { static: true })
+  @ViewChild(TableComponent, { static: false })
```

Comment:
Default value for `static` is `false`, so we can remove it:
```suggestion
  @ViewChild(TableComponent)
```